### DISCUSSION TITLE
New projects return SSL SYSCALL error: EOF detected when connecting from Postico

### DISCUSSION THREAD

**Discussion by loama** [supabase/supabase#6077] [Questions] [answered]
Not sure what's happening but I have to projects a couple of months old for which I use Postico (mac app) to explore the database and it works perfectly.

But I just created a new one and it failed to connect while returning `SSL SYSCALL error: EOF detected`
It does work from the dashboard and from PgAdmin though.

**Comment by ghost** [comment]
Not a solution, but some additional information. I'm getting the same problem on a new Supabase project - Postico simply won't connect with the `SSL SYSCALL error: EOF detected`. I can however connect from plain old `psql`, _but_, running `SHOW ALL` will cause the same error. Running something less intense like `SHOW SERVER_VERSION` returns successfully.

From what I've read about the issue in general, it's either related to resource on the Postgres server to the an issue with the SSL configuration. Given it's possible to establish an SSL connection from certain clients, it's unlikely to be an SSL issue and is therefore more like caused by resource on the server itself.

My _guess_, as far as Postico is concerned, is that it's attempting to do additional _stuff_ during the connection process which is tripping the limit. I've tried to adjust the settings in Postico to get it to connect, but have had no joy 😞 

I remember reading a blog post a few days ago about how Supabase had managed to squeeze all of the services onto small virtual servers to allow them to offer the free-tier - it's quite an impressive achievement. I suspect moving to paid plan would mean more resource and would resolve this particular issue.

**Reply by jaykaycodes** [reply]
@altcmdio Looks like a paid plan does not resolve this particular issue. Having the exact same problem after upgrading/migrating my production (pro tier) db.

**Comment by darora** [comment, marked as answer]
We've tracked down the issue to an extension that we use, and will be tracking resolution at https://github.com/supabase/supautils/issues/24

**Comment by DanMossa** [comment]
@darora Hey! It looks like that issue is merged in but I'm still having the error via postico.

**Reply by allanhortle** [reply]
Same here.

**Reply by darora** [reply]
Most projects have been updated now - let us know via a support ticket if you're still facing issues.

**Reply by keslert** [reply]
@darora Still getting the error when I tried to connect via Postico this morning.

**Reply by lucsky** [reply]
Created a new project two days ago and got the dreaded `SSL SYSCALL error` when trying to connect with Postico2. I'm glad I found this thread and that the issue is being monitored, but it still does occur.

**Reply by lucsky** [reply]
@darora Still getting the error, I have sent a ticket request and have been asked to rather respond here. So which is it?

**Reply by darora** [reply]
Sorry about that; it was an internal miscommunication on our end. Your project has been fixed (and all new projects as of the 6th of May are fine as well; we'll be making another pass over older projects next week).

I'm going to lock this discussion - support is still the right venue for this issue.
